### DISCUSSION TITLE
Apply to Unifi access clips

### DISCUSSION THREAD

**Discussion by limacub777** [ep1cman/unifi-protect-backup#128] [Ideas]
It would be great if this solution could be applied to Unifi access door unlock recordings and have a backup.
After manual or auto retention period all recordings get deleted. Hopefully the authors could develop a script for such scenario

**Comment by ep1cman** [comment]
I am afraid I don't have access to "Unifi access" so cannot develop such a feature, but if anyone in the community can I would gladly accept a pull request with the feature

**Reply by limacub777** [reply]
I should definitely be able to provide you access to "Unifi Access" I'm not familiar with pull request, I'll try to submit one

Thanks
